hnn_core/tests/test_optimization.py

```diff
@@ -136,3 +136,91 @@ def test_optimize_evoked():
 
     # static drive params should remain constant
     assert drive_static_params_opt == drive_static_params_orig
+
+
+def test_optimize_evoked_selectedDrive():
```

Comment:
another [style convention](https://peps.python.org/pep-0008/#function-and-variable-names). In Python, function names are usually snake_case rather than CamelCase. This is more Matlab style :)